From e3d1f5505cb5aac2201ca2cd6828356cc7288553 Mon Sep 17 00:00:00 2001
From: ahnet-69 <syedmusicrealise@gmail.com>
Date: Sun, 24 Dec 2023 13:33:33 +0500
Subject: [PATCH] Remove definations of ro.adb.secure

---
 core/main.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index eb12c67..2673354 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -375,10 +375,6 @@ ifneq (,$(user_variant))
   # Target is secure in user builds.
   ADDITIONAL_SYSTEM_PROPERTIES += ro.secure=1
 
-  ifeq ($(user_variant),user)
-    ADDITIONAL_SYSTEM_PROPERTIES += ro.adb.secure=1
-  endif
-
   # Disable debugging in plain user builds.
   enable_target_debugging :=
 
-- 
2.34.1

